src/controllers/queryRunner.ts

```diff
@@ -561,6 +561,27 @@ export default class QueryRunner {
         return headers;
     }
 
+    public async copyHeaders(batchId: number, resultId: number): Promise<void> {
+        let headers = this.getColumnHeaders(batchId, resultId, {
+            fromCell: 0,
+            fromRow: 0,
+            toCell: this.batchSets[batchId].resultSetSummaries[resultId]
+                .columnInfo.length,
+            toRow: this.batchSets[batchId].resultSetSummaries[resultId]
+                .columnInfo.length,
+        });
+        let headerString = headers.join("\t");
+        let oldLang: string;
+        if (process.platform === "darwin") {
```

Comment:
can you add a comment here for why there's special casing for mac?

Reply:
I'm not sure why it's special casing for mac, this logic was copied from the copy results portion of the code